constellation: Optimize tracing of maybe_close_random_pipeline

jschwe opened the pull request:
Most of the time this function does nothing and returns immediately, so there isn't really a point in tracing it.
If one wanted to know when a pipeline is closed, one could add a trace point **after** the early return.
However, during review it was suggested that we currently don't need this information and thus just remove the trace.

Testing: Not required.

mrobinson reviewed (changes requested):
I think I would prefer not to trace this function at all. It's just used for testing, so we shouldn't be too worried about the performance.

jschwe committed:
tracing: Don't trace maybe_close_random_pipeline

Most of the time this function does nothing
and returns immediately,
so there isn't really a point in tracing it.
If one wanted to know when a pipeline is closed,
one could add a trace point **after** the early return.
However during review it was suggested that we currently don't need this
information and thus just remove the trace.

Signed-off-by: Jonathan Schwender <schwenderjonathan@gmail.com>

jschwe commented:
@mrobinson I came back to this PR, updated the description and just removed the trace point as you suggested.
